code/game/objects/structures/gym.dm

```diff
@@ -20,6 +20,31 @@
 		L.add_mood_event("exercise", /datum/mood_event/exercise)
 		L.apply_status_effect(/datum/status_effect/exercised)
 
+/obj/structure/punching_bag/wrench_act(mob/living/user, obj/item/tool)
```

Comment:
All of these should support `_act_secondary` as well--we want to formalize *de*construction being on right-click, not left-click.